features/sentinel/repository/pom.xml

```diff
@@ -59,6 +59,8 @@
                                 <feature>scv-api</feature>
                                 <feature>scv-shell</feature>
                                 <feature>scv-jceks-impl</feature>
+                                <feature>cxf-commands</feature>
```

Comment:
Do we need `cxf-commands` ?    We also need to add `opennms-health-rest-service`  and new dependency in minion/repository/pom.xml as well.

Reply:
- Included Karaf features in () under org.apache.karaf.features.cfg so there are no missing dependencies at boot.
- Rearrange health rest features to include required features.
- cfx-commands was already included in minion features (minion/resources/pom.xml)